task.sql

```diff
@@ -38,3 +38,18 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+
+--changeset confot:1 labels:0.0.2
+CREATE TABLE Users (
+    ID INT AUTO_INCREMENT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50),
+    PRIMARY KEY (ID)
+);
+--rollback DROP TABLE Users;
+
+--changeset confot:2 labels:0.0.3
+CREATE INDEX Email ON Users (Email);
+--rollback DROP INDEX Email ON Users;
```

Comment:
Додай EOF (End Of File)
```suggestion
--rollback DROP INDEX Email ON Users;

```